docs/source/tutorial/first.rst

```diff
@@ -7,13 +7,13 @@ First Optimization
 Quadratic Function Example
 --------------------------
 
-Let us try very simple optimization in IPython shell.
+Usually, Optuna is used as a hyper-parameter optimizer, but as an example, let us try a direct optimization in IPython shell.
```

Comment:
Is `hyper-parameter optimizer` a common phrase?. Maybe it is just me but how about something along `Usually, Optuna is used to optimize hyper-parameters, ...`.

Reply:
Agreed, and updated.